cmake/FindWdk.cmake

```diff
@@ -60,13 +60,14 @@ get_filename_component(WDK_ROOT ${WDK_ROOT} DIRECTORY)
 
 message(STATUS "WDK_ROOT: " ${WDK_ROOT})
 message(STATUS "WDK_VERSION: " ${WDK_VERSION})
-
+enable_language（ASM_MASM）# for masm
```

Comment:
`enable_language` should be called in a `cmakelists.txt` of the project that needs it. Not in this module.